src/app/router/Prompt/Prompt.tsx

```diff
@@ -60,6 +60,16 @@ function Prompt() {
     <Providers>
       <HashRouter>
         <Routes>
+          <Route
+            index
```

Comment:
Shouldn't this be path="/"? See https://reactrouter.com/en/main/route/route#index

Now there are two path elements on the same route, I'm not sure this is a good idea?

Reply:
https://reactrouter.com/en/main/start/concepts#index-routes

afaik that route navigates user to appropriate screen after screen is unlocked. eg. getPublicKey. before that route was inside <Route path="/"> which has RequireAuth guard. 

as for nostr we handle unlock screen explicitly and to avoid two popups for unlock screen. i kept it outside the main route ("/").  so yet it will work same as before. along with routes that don't require Auth guard